.github/workflows/prepare-release.yaml

```diff
@@ -166,6 +154,24 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.BOT_SDK_JS_FOR_DOCS_REPO_PR }}
 
+      - name: "Run Continuous Integration" # It should trigger on push but sometimes doesn't
+        run: |
+          gh workflow run continuous-integration.yaml --ref ${{ needs.bump-version.outputs.release-branch }}
+        env:
+          GH_TOKEN: ${{ secrets.BOT_SDK_JS_FOR_DOCS_REPO_PR }}
+
+      - name: "Set New Version and Reset Release Notes"
+        run: |
+          python .pipeline/scripts/set-release-versions.py --version ${{ needs.bump-version.outputs.new-version }}
+          git add .
+          git commit -m "Update to version ${{ needs.bump-version.outputs.new-version }}"
+          
+          # Reset release notes for next version
+          cp .pipeline/scripts/release_notes_template.md docs/release-notes/release_notes.md
+          git add docs/release-notes/release_notes.md
+          git commit -m "Reset release notes"
+          git push
```

Comment:
(Question/Minor)

Is it correct to create a PR before pushing actual changes? Or am I reading the workflow wrongly?

Reply:
I pushed in the step before already, there are 2 pushes total